README.md

```diff
@@ -84,6 +87,16 @@ type TdxConfigSpec struct {
   // +optional
   KbsTdxConfigMapName string `json:"kbsTdxConfigMapName,omitempty"`
 }
+
+// KbsMountDirConfigSpec defines the configuration needed for mounting a local directory into the trustee file system
+type KbsMountDirConfigSpec struct {
```

Comment:
Since it's mounting a secret, should this be named accordingly to reflect that ?
For example `type KbsLocalCertCacheSpec` ?